debian/upstream/metadata

```diff
@@ -0,0 +1,5 @@
+Name: cloud-init
+Contact: Josh Powers <josh.powers@canonical.com>
```

Comment:
Should we make this a mailing list?

Reply:
I think it's a good idea; is there one more appropriate than ubuntu-server@lists.ubuntu.com?